ssh/tailssh/tailssh.go

```diff
@@ -226,6 +240,7 @@ func (srv *server) sshPolicy() (_ *tailcfg.SSHPolicy, ok bool) {
 		return pol, true
 	}
 	if debugPolicyFile != "" {
+		srv.logf("reading debug SSH policy file: %v", debugPolicyFile)
```

Comment:
"error reading ..."

(I like the word "error" in errors so it's easier to consistently grep for them)

Reply:
not an error, just log that it's reading the debug file